data_steward/constants/bq_utils.py

```diff
@@ -50,7 +50,8 @@
 
 # Dataset name formats
 VALIDATION_DATASET_FORMAT = 'validation_{}'
-VALIDATION_DATASET_REGEX = 'validation_\d{8}'
+VALIDATION_DATASET_REGEX = 'validation_(\d{8})'
```

Comment:
Does the addition of brackets lead to any changes in the function that was previously using this regex?

Reply:
This bracket only adds a new group in the pattern match and doesn't affect the pattern match. I will double-check though.